activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -163,9 +163,37 @@ def test_nilable_default_scope_with_all_queries_runs_on_update
     assert_no_match(/AND$/, update_sql)
   end
 
+  def test_scoping_with_where_without_all_queries_and_default_scopes_with_and_without_all_queries_on_update
+    Mentor.create!
+    dev = DeveloperWithIncludedMentorDefaultScopeNotAllQueriesAndDefaultScopeFirmWithAllQueries.create!(name: "Josh")
+    DeveloperWithIncludedMentorDefaultScopeNotAllQueriesAndDefaultScopeFirmWithAllQueries
+      .where("1=1")
```

Comment:
Is this needed just to have a scope? Could you do this?

```suggestion
      .all
```